######################################################################
# Hexapod Parameters
######################################################################

hexapod_type: large_hexapod
move_to_start: false

imu_compensation: false
auto_compensation: true
pitch_amplitude: 0.025
tilt_amplitude: 0.025
manual_compensation: true

gait_type: wave_gait

######################################################################


stance_leg_yaws: [0.785, 0, -0.785]
yaw_limits: [0.523, 0.523, 0.523]
knee_limits: [0.872, 2.791]
hip_limits: [-0.436, 1.396]

joint_max_angular_speeds: [1.0, 0.4, 0.4]

dynamixel_interface: true

######################################################################
# Walk Controller Parameters
######################################################################

step_frequency: 0.2
step_clearance: 0.06
body_clearance: 0.4
leg_span_scale: 0.6

max_acceleration: 0.1
max_curvature_speed: 0.4

step_curvature_allowance: 0.7

interface_setup_speed: 0.5

######################################################################
######################################################################
